.github/workflows/gh-actions.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - run: sudo apt update
 # TODO: update checkers to current versions available in ubuntu 22.04
 #      - run: sudo apt install doxygen clang-format-10 cppcheck pylint python-serial
+      - run: sudo apt install pylint
```

Comment:
I would keep the rest in the comment. I suspect they will be needed.

Reply:
I was thinking they would the added back as needed, also some of their names are different in ubuntu-22.04.

Reply:
The comment shows the missing tools, regardless of the new name. If we delete it, we don't know which tools are still not added back. Instead, we should delete them one by one when they are added back.